backend/app.js

```diff
@@ -24,7 +28,7 @@ mongoose.connect(process.env.DB_CONNECTION, {useNewUrlParser: true, useUnifiedTo
 	.then(() => console.log('mongoDB Atlas Connected!'))
```

Comment:
connection to db missing, .then and .catch will give error

Reply:
uuuh, no its not... thats just github compressing lines 11-23. Did you check the line numbers????